chapters/handling-data.tex

```diff
@@ -30,59 +34,62 @@
+for using field data ethically and responsibly to publish research findings.
 \end{fullwidth}
 
 %------------------------------------------------
 
 \section{Protecting confidence in development research}
 
 The empirical revolution in development research
-\index{transparency}\index{credibility}\index{reproducibility}
+  \index{transparency}\index{credibility}\index{reproducibility}
 has led to increased public scrutiny of the reliability of research.\cite{rogers_2017}
-Three major components make up this scrutiny: \textbf{replicability}.\cite{duvendack2017meant}, \textbf{transparency},\cite{christensen2018transparency} and \textbf{credibility},\cite{ioannidis2017power}.
-Replicability is one key component of transparency.
-Transparency is necessary for consumers of research
-to be able to tell the quality of the research.
+Three major components make up this scrutiny: \textbf{reproducibility}.\cite{duvendack2017meant}, \textbf{transparency},\cite{christensen2018transparency} and \textbf{credibility},\cite{ioannidis2017power}.
+Reproducibility is one key component of transparency.
+Transparency is necessary for consumers of research products
+to be able to determine the quality of the research process and the value of the evidence.
 Without it, all evidence of credibility comes from reputation,
 and it's unclear what that reputation is based on, since it's not transparent.
 
-Development researchers should take these concerns seriously.
-Many development research projects are purpose-built to cover specific questions,
-and may utilize unique data or small samples.
-This approach opens the door to working with the development community
-to answer both specific programmatic questions and general research inquiries.
-However, the data that researchers utilize have never been reviewed by anyone else,
+Development researchers should take these concerns particularly seriously.
+Many development research projects are purpose-built to address specific questions,
+and often use unique data or small samples.
+This approach opens the door to working closely with the broader development community
+to answer specific programmatic questions and general research inquiries.
+However, almost by definition,
+primary data that researchers use for such studies has never been reviewed by anyone else,
 so it is hard for others to verify that it was collected, handled, and analyzed appropriately.
-Maintaining confidence in research via the components of credibility, transparency, and replicability
-is the most important way that researchers can avoid serious error,
-and therefore these principles are not by-products but core components of research output.
+Maintaining confidence in research via the components of credibility, transparency, and reproducibility
+is the most important way that researchers using primary data can avoid serious error,
+and therefore these are not by-products but core components of research output.
```

Comment:
When reading through I made a note that this sections needs to be done more explicit. I think Luiza's version is more on point, but maybe it can be improved even more.

I think it should be made its own paragraph to emphasize it a little. It currently disappears a little in the end of such a long paragraph in the end of a section.

Reply:
Tried new wording in a05d9aba184ba851b9f25d48a0ab26d8dc8bcaa2. Didn't want to make a separate paragraph at this point.

Reply:
Sorry, I don't see how these changes affect the content of the sentence.

Reply:
Ok, tried again based on your suggestion in a47e7867df40bf1af9ef4f3c5be393fe7eeceea9

Reply:
That's a wrap on the chapter!!